src/host/pico_platform/include/pico/platform.h

```diff
@@ -20,7 +20,7 @@
 extern "C" {
 #endif
 
-#define __not_in_flash(grup)
+#define __not_in_flash(grpup)
```

Comment:
`grpup`? Is that deliberate, or is it a typo for `group` ?

Reply:
Typo, but harmless since it's an empty macro. Fixed